### ISSUE TITLE
Platform Modes Save

### ISSUE THREAD

**Issue by kelunik** [coolcrowd/object-service#239]
![image](https://cloud.githubusercontent.com/assets/2743004/13819467/cb3dd2d0-eb99-11e5-9f2b-62f80a5579bd.png)

**Comment by kelunik** [comment]
Image was for read, see #240

**Comment by kelunik** [comment]
Trace was for read, see #240

**Comment by LeanderK** [comment]
i think this was fixed with #238, can you reproduce it?

**Comment by kelunik** [comment]
Not fixed in current `master`:

```
{
  "code": "internalServerError",
  "detail": "Key 3 is not unique in Result for +------+--------------------+---------------------+----------+\n|mode  |experiments_platform|max                  |platform  |\n+------+--------------------+---------------------+----------+\n|normal|                  11|2016-03-17 13:01:38.0|dummydummy|\n|answer|                  11|2016-03-17 13:01:45.0|dummydummy|\n+------+--------------------+---------------------+----------+"
}
```

When changing from one mode to another mode.
